apps/nuxt3-ssr/interfaces/types.ts

```diff
@@ -1,5 +1,8 @@
-import type { IColumn } from "../../metadata-utils/dist";
-import type { INode } from "../../tailwind-components/types/types";
+import type {
+  IDocumentation,
+  IFile,
+  INode,
+} from "../../tailwind-components/types/types";
```

Comment:
If these are used across apps, should they be moved to a more generic interface lib?